config.m4

```diff
@@ -944,9 +944,26 @@ EOF
     CFLAGS="-Wall -pthread $CFLAGS"
     LDFLAGS="$LDFLAGS -lpthread"
 
-    if test "$SW_OS" != "MAC"; then
+	dnl Check should we link to librt
+	OS_SHOULD_HAVE_LIBRT=1
+
+	if test "$SW_OS" = "MAC"; then
+		OS_SHOULD_HAVE_LIBRT = 0
```

Comment:
This breaks the build on macOS

https://github.com/swoole/swoole-src/pull/5194